lib/predictions/adaptive_last_predictor.rb

```diff
@@ -9,7 +9,7 @@ def self.suggestions
 
   def self.predict(history)
     return nil if !history
-    return "p" if history.empty?
+    return "r" if history.empty?
```

Comment:
predictors should predict what the opponents first move will be not how to counter it. That is the reason for this change.